docusaurus.config.js

```diff
@@ -97,6 +97,18 @@ const config = {
           // },
         ],
       },
+      metadata: [
+        {name: 'og:title', content: 'Aragon Developer Portal'},
+        {name: 'og:type', content: 'documentation'},
+        {name: 'og:url', content: 'https://devs.aragon.org'},
+        {name: 'og:image', content: 'https://devs.aragon.org'},
```

Comment:
Isn't it redundant to have a "learn more" link pointing to the page itself?

Reply:
What's the "learn more" link you're mentioning? This is for social sharing.

I'll remove the `og:image` as of now since I'm still waiting to get it.